exporter/fileexporter/factory.go

```diff
@@ -63,7 +69,13 @@ func createMetricsExporter(
 	fe := exporters.GetOrAdd(cfg, func() component.Component {
 		return &fileExporter{path: cfg.(*Config).Path}
 	})
-	return exporterhelper.NewMetricsExporter(cfg, params.Logger, fe.Unwrap().(*fileExporter).ConsumeMetrics)
+	return exporterhelper.NewMetricsExporter(
+		cfg,
+		params.Logger,
+		fe.Unwrap().(*fileExporter).ConsumeMetrics,
+		exporterhelper.WithStart(fe.Start),
+		exporterhelper.WithShutdown(fe.Shutdown),
```

Comment:
@sincejune here is where you passed the instance start/shutdown instead of the SharedComponent start/shutdown

Reply:
@bogdandrutu Thanks! Now I understand.